_posts/2021-01-07-debezium-1-4-final-released.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  Debezium 1.4.0.Final Released
+date:   2021-01-07
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: ccranfor
+---
+
+I am pleased to announce the release of Debezium *1.4.0.Final*!
+
+This release concludes the major work put into Debezium over the last three months.
+Overall, the community fixed https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20in%20(1.4.0.Final%2C%201.4.0.Alpha1%2C%201.4.0.Beta1%2C%201.4.0.CR1)%20ORDER%20BY%20issuetype%20DESC%2C%20updated%20DESC%2C%20priority%20DESC[117 issues] during that time, including the following key features and changes:
+
+* New link:/documentation/reference/connectors/vitess.html[Vitess] connector
+* Fine-grained Selection of Snapshotted Tables
+* PostgreSQL `Snapshotter` completion hook
+* Distributed link:/blog/2020/12/16/distributed-tracing-with-debezium/[Tracing]
+* MySQL support for _create_ or _read_ records emitted during snapshot
+* Many Oracle link:/documentation/reference/connectors/oracle.html#_logminer[Logminer adapter] improvements
```

Comment:
LM support was new to begin with in 1.4, right?

Reply:
No, we added it late in 1.3 but continued to stablize & improve it during 1.4.